templates/archive.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Weekly Community News - {{date}}
+date: {{date}}
+tags: {{tags}}
+youtube: {{youtube}}
+podcast: {{podcast}}
+github: https://github.com/kjaymiller/Python-Community-News/blob/main/app/content/{{date}}.md
```

Comment:
New repo isn't populated yet, but we'll want to change this to reference https://github.com/Python-Community-News/Site once it is.

Reply:
TBH I could remove `archive.md` at this point (it's there for historical) actually I will do that